testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/DeployTestCase.java

```diff
@@ -571,4 +572,142 @@ public void testEnableWrongDeployments() throws Exception {
                     after, is(before));
         }
     }
+
+    /**
+     * Testing disabling of non-deployed application deployments.
+     * Verify if status of application deployments hasn't change.
+     * Verify error message.
+     *
+     * @throws Exception
+     * @see <a href="https://issues.jboss.org/browse/WFCORE-3566">WFCORE-3566</a>
+     */
+    @Test
+    public void testDisableWrongDeploymentWithServerGroups() throws Exception {
```

Comment:
ServerGroups are out of scope. Could you rename the methods. Apply to all of them I think.

Reply:
I mixed two classes. Thank you.